### PR TITLE
[PIM-6886] Update docs for 2.0

### PR DESCRIPTION
**Pull request by pierallard**

### CODE REVIEW

File: design_pim/guides/how_to_customize_menu.rst [hunk 1]
```diff
@@ -19,14 +19,14 @@ To add an element at the root of the tree you can reuse the tab module provided
         config:
             title: pim_menu.item.import_profile        # You can define a translation key for the tab name
 
-After running the command ``bin/console pim:install:asset`` your new item should appear at the root of the menu
+After running the command ``rm -rf var/cache; bin/console pim:install:asset`` your new item should appear at the root of the menu.
```
Comment (juliensnz) [inline design_pim/guides/how_to_customize_menu.rst]: we should also run yarn webpack?

File: design_pim/guides/add_a_tab_to_form.rst [hunk 2]
```diff
@@ -83,15 +88,15 @@ Now that we have our extension loaded in our form, we can add some logic into it
     /*
      * /src/Acme/Bundle/EnrichBundle/Resources/public/js/product/form/packages.js
      */
-    define(['underscore', 'pim/form', 'text!pim/template/product/tab/packages'],
-        function (_, BaseForm, template) {
+    define(['underscore', 'oro/translator', 'pim/form', 'pim/template/product/tab/packages'],
+        function (_, __, BaseForm, template) {
```
Comment (phaseinducer) [inline design_pim/guides/add_a_tab_to_form.rst]: :heart:

File: design_pim/guides/how_to_customize_menu.rst (hunk 1 above)
Comment (phaseinducer) [reply design_pim/guides/how_to_customize_menu.rst]: Not if you modified only the yml conf. But I don't think you need to dump assets.

File: design_pim/guides/how_to_customize_menu.rst [hunk 3]
```diff
@@ -39,7 +39,29 @@ Now if you want to add an element inside the menu, you can use the item module:
             title: pim_menu.item.import_profile        # You can define a translation key for the item name
             to: pim_importexport_import_profile_index  # The route to redirect to
 
-After running the command ``bin/console pim:install:asset`` your new item should appear in the menu
+After running the command ``rm -rf var/cache; bin/console pim:install:asset`` your new item should appear in the menu.
```
Comment (phaseinducer) [inline design_pim/guides/how_to_customize_menu.rst]: No need to dump the assets.

File: design_pim/guides/how_to_customize_menu.rst [hunk 4]
```diff
@@ -39,7 +39,29 @@ Now if you want to add an element inside the menu, you can use the item module:
...
+
+    acme-custom-page-breadcrumbs:
+        module: pim/common/breadcrumbs
+        parent: acme-custom-page
+        targetZone: breadcrumbs
+        config:
+            tab: acme-custom-root-element
+            item: acme-custom-sub-element
+
+After running the command ``rm -rf var/cache; bin/console pim:install:asset`` the menu will be highlited when you will open your custom page.
```
Comment (phaseinducer) [inline design_pim/guides/how_to_customize_menu.rst]: idem

File: design_pim/guides/add_a_tab_to_form.rst (hunk 2 above)
Comment (pierallard) [reply design_pim/guides/add_a_tab_to_form.rst]: Just for you @phaseinducer

**Pull request merged**